### DISCUSSION TITLE
Country, Currency and Timezone

### DISCUSSION THREAD

**Discussion by jebento** [kimai/kimai#3312] [General]
Hi, 
Country, Currency and Timezone, where does this information come from?
Not found in table.

Att. Eduardo

**Comment by kevinpapst** [comment]
What exactly do you want to know?

**Comment by jebento** [comment]
Hi,

These fields are already filled in.
Example: Field Country, there is Brazil, I would like to know where it will
get this information.
Is there any table registered this information

Em ter., 17 de mai. de 2022 às 16:49, Kevin Papst ***@***.***>
escreveu:
